scripts/refresh_project.sh

```diff
@@ -67,7 +67,7 @@ set -e
 if [ $return_code -ne 0 ]; then
   pip install uv
 fi
-uv sync
+uv sync --no-cache
```

Comment:
```suggestion
uv sync --no-cache --link-mode=copy
```

`uv` throws a warning at the end saying that because `/tmp` and `/home/cdsw` are different filesystems (the latter is a mount iirc), it can't symlink packages and has to copy them instead. `--link-mode=copy` makes that explicit and disables the warning.